config/locales/en/placements/schools/placements.yml

```diff
@@ -2,12 +2,25 @@ en:
   placements:
     schools:
       placements:
-        not_entered: Not entered
+        terms:
+          autumn: Autumn
+          spring: Spring
+          summer: Summer
+        not_known_yet: Not known yet
+        window_date: "%{start_month} to %{end_month}"
         index:
           page_title: Placements
           placements: Placements
           subject: Subject
           mentor: Mentor
-          start_date: Start date
+          placement_window: Placement window
           status: Status
           not_entered: Not entered
+        show:
+          attributes:
+            placements:
+              school_level: School level
+              subject: Subject
+              mentor: Mentor
+              placement_window: Placement window
```

Comment:
Ditto – https://github.com/DFE-Digital/school-placements-beta-prototype/pull/56

Reply:
resolved in [5a5c21a](https://github.com/DFE-Digital/itt-mentor-services/pull/325/commits/5a5c21a937ac7f819a6ad8309c14c9c61b31c749)